.github/workflows/test-gcp-dm.yml

```diff
@@ -118,9 +120,121 @@ jobs:
 
       - name: Delete GCP Deployment Manager deployment
         if: always()
-        working-directory: ${{ env.WORKING_DIR }}
+        working-directory: ${{ env.TEST_ENVS_DIR }}
         run: |
           DEPLOYMENT=${{env.GCP_DEPLOYMENT_NAME}}
           PROJECT_NAME=$(gcloud config get-value core/project)
           PROJECT_NUMBER=$(gcloud projects list --filter="${PROJECT_NAME}" --format="value(PROJECT_NUMBER)")
           ./delete_gcp_env.sh $PROJECT_NAME $PROJECT_NUMBER $DEPLOYMENT
+
+  # Test a GCP Deployment Manager deployment using a Service Account
+  gcp_dm_sa:
```

Comment:
this job tests the code path that uses a service account json 
https://github.com/elastic/cloudbeat/blob/e8309ef67d6bb49119a9464ff00ae0b9bf815a44/internal/resources/providers/gcplib/auth/credentials.go#L63